numba/tests/test_np_functions.py

```diff
@@ -2448,6 +2452,56 @@ def a_variations():
 
         self.assertIn("Cannot np.flip on UniTuple", str(raises.exception))
 
+    def test_logspace_basic(self):
+        pyfunc = logspace
+        cfunc = jit(nopython=True)(pyfunc)
+
+        self.assertPreciseEqual(pyfunc(1, 60), cfunc(1, 60))
+        self.assertPreciseEqual(pyfunc(-60, 1), cfunc(-60, 1))
+        self.assertPreciseEqual(pyfunc(-1, -60), cfunc(-1, -60))
+
+        self.assertPreciseEqual(pyfunc(1.0, 60.0), cfunc(1.0, 60.0))
+        self.assertPreciseEqual(pyfunc(-60.0, -1.0), cfunc(-60.0, -1.0))
+        self.assertPreciseEqual(pyfunc(-1.0, 60.0), cfunc(-1.0, 60.0))
+
+        self.assertPreciseEqual(pyfunc(0.0, np.e), cfunc(0.0, np.e))
+        self.assertPreciseEqual(pyfunc(0.0, np.pi), cfunc(0.0, np.pi))
+
+    def test_logspace_with_num_basic(self):
+        pyfunc = logspace
+        cfunc = jit(nopython=True)(pyfunc)
+
+        self.assertPreciseEqual(pyfunc(1, 50, num=20), cfunc(1, 50, num=20))
```

Comment:
Perhaps consider factoring this out as:
```python
def check(start, stop, num):
    self.assertPreciseEqual(pyfunc(start, stop, num=num), cfunc(start, stop, num=num))
    self.assertPreciseEqual(pyfunc(start, stop, num), cfunc(start, stop, num=num))
```
and using that throughout? Also note the comment above about checking the default args by having 2 and 3 arg variations.